pkg/commands/git_commands/rebase.go

```diff
@@ -122,15 +122,24 @@ func (self *RebaseCommands) InteractiveRebaseBreakAfter(commits []*models.Commit
 	}
 
 	todo = append(todo, TodoLine{Action: "break", Commit: nil})
-	return self.PrepareInteractiveRebaseCommand(sha, todo, true).Run()
+	return self.PrepareInteractiveRebaseCommand(sha, todo, true, false).Run()
+}
+
+func (self *RebaseCommands) EditRebase(branchRef string) error {
+	commands := []TodoLine{{Action: "break"}}
+	return self.PrepareInteractiveRebaseCommand(branchRef, commands, false, true).Run()
 }
 
 // PrepareInteractiveRebaseCommand returns the cmd for an interactive rebase
 // we tell git to run lazygit to edit the todo list, and we pass the client
 // lazygit a todo string to write to the todo file
-func (self *RebaseCommands) PrepareInteractiveRebaseCommand(baseShaOrRoot string, todoLines []TodoLine, overrideEditor bool) oscommands.ICmdObj {
+func (self *RebaseCommands) PrepareInteractiveRebaseCommand(baseShaOrRoot string, todoLines []TodoLine, overrideEditor bool, prepend bool) oscommands.ICmdObj {
```

Comment:
We've got quite a few positional arguments here now. Let's create a PrepareInteractiveRebaseCommandOpts struct which can be passed as the single argument to this method.